app/web/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -243,10 +243,10 @@ export default function HostRequestSendField({
       </div>
       <div className={classes.container}>
         <TextField
-          defaultValue={""}
+          defaultValue={isRequestClosed ? REQUEST_CLOSED_MESSAGE : ""}
           disabled={isRequestClosed}
           fullWidth
-          label={isRequestClosed ? REQUEST_CLOSED_MESSAGE : "Message"}
+          label={!isRequestClosed ? "Message" : ""}
```

Comment:
I think this would cause accessibility issues for non-visual users, as ideally they should be able to find the field as disabled with the "Message" label when the request is closed, which would not be possible if we render an empty string in that case.

The best way to solve this without changing the visible UI might be via an `aria-label` with the same label value.

Reply:
I added an aria-label.

Reply:
Sorry if I wasn't being clear - it's better to stick with the same "Message" label since the input box hasn't changed its purpose, but disabled when the request is closed.

Reply:
Ah I see, got it 😅 I updated it, so the aria-label is always "Message"